packages/design-system/src/components/DateField/MultiInputDateField.stories.tsx

```diff
@@ -4,7 +4,7 @@ import MultiInputDateField from './MultiInputDateField';
 import type { Meta, StoryObj } from '@storybook/react';
 
 const meta: Meta<typeof MultiInputDateField> = {
-  title: 'Components/MultiInputDateField',
+  title: 'Components/Multi Input Date Field',
```

Comment:
I don't love that this creates new snapshots, and that it isn't the name of the component exactly.

Reply:
That would go for all changes of this type.

Reply:
I guess we could have a larger discussion about the naming in storybook. I can move that convo to slack to rope everyone in. There were two competing naming conventions, so I picked the one that seemed more legible

Reply:
Update: We've landed on the stories/folders in the components section to have the exact same name as their corresponding component (`PascalCase`) but non-component in other sections would use `Sentence case`

Reply:
I'm going to make that update in a follow-up PR

Reply:
I've reverted all the naming to what it was before, and I'll create a new PR to make them consistent within the components section